--- origsrc/libgalago-0.5.2/libgalago/Makefile.am	2006-05-17 04:01:56.000000000 -0500
+++ src/libgalago-0.5.2/libgalago/Makefile.am	2007-06-20 06:56:43.203125000 -0500
@@ -58,7 +58,7 @@
 	$(PACKAGE_LIBS)
 
 libgalago_la_LDFLAGS = \
-	-version-info $(LT_CURRENT):$(LT_REVISION):$(LT_AGE)
+	-version-info $(LT_CURRENT):$(LT_REVISION):$(LT_AGE) -no-undefined
 
 galago-marshal.h: galago-marshal.list
 	$(GLIB_GENMARSHAL) --prefix=$(MARSHAL_PREFIX) galago-marshal.list \
